source/includes/_01-config.html.md

```diff
@@ -106,7 +98,10 @@ customers have a direct commercial relationship with the data provider.
 </table>
 
 <aside>
-  TODO: More info on PassFort credits
+  PassFort credits are fungible tokens which are purchased by PassFort's
```

Comment:
"fungible" is an amazing word! But a bit complicated. We can remove it in this sentence because we're already saying "spent with any provider."

I'd also break down the thoughts into 2 sentences: "PassFort credit is purchased by PassFort's customers. It's used to pay for checks when PassFort is acting as the data provider reseller."